research/src/pages/tabs.proposal.mdx

```diff
@@ -0,0 +1,283 @@
+---
+menu: Proposals
+name: Tabs (Editor's Draft)
+path: /components/tabs
+pathToResearch: /components/tabs.research
+showInMenu: false
+---
+
+# Tabs Component Specification
+
+## Overview <a href="#overview" id="overview"></a>
+
+*Tabs* are a set of layered sections of content that display one panel of content at a time. Each tab panel has an associated tab element, that when activated, displays the panel. The list of tab elements is arranged along one edge of the currently displayed panel.
+
+### Background <a href="#background" id="background"></a>
+
+_Relevant historical or background information, related existing issues, etc._
+
+### Use Cases <a href="#use-cases" id="use-cases"></a>
+
+- Steve is researching for an upcoming vacation for him and his family. He opens his web browser and opens a new tab to make a rental car reservation and another for his flight information. Steve can quickly switch back and fourth between tabs to get the information he needs for the rental car reservation.
+
+- Monika visits a website to learn how to fix her leaky sink. The website displays a vertical tab interface with each step. Monika can jump ahead steps or go back to previous steps by selecting various tabs.
```

Comment:
I'm not suggesting we change this before merging but I feel like we can add/find better use case docs in OS guides maybe?  This sounds awfully like someone will say "that's a wizard" not tabs or something

Reply:
Another idea for an example are mobile patterns to indicate sections: I.e. the Twitter app Explore page

Reply:
https://github.com/WICG/open-ui/issues/189